index.html

```diff
@@ -0,0 +1,198 @@
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" href="css/reset.css">
+        <link rel="stylesheet" href="css/mobile-style.css">
+        <link rel="stylesheet" media="(min-width: 768px)" href="css/tablet-style.css" />
+        <link rel="stylesheet" media="(min-width: 1024px)" href="css/desktop-style.css" />
+        <title>Next Web Clone</title>
+    </head>
+    <body>
+        <nav id="navbar">
+            <div id="nav-icons">
+                <div id="navbar-flex-container">
+                    <div>
+                        <img src="assets/tnw-logo.png" alt="logo" id="logo-img" />
+                    </div>
+                    <div id="navbar-links">
+                        <div id="top-nav-links">
+                            <div id="top-nav-links-left">
+                                <div class="inline-list"><a href="#" class="normal-link ">News</a></div>
+                                <div class="inline-list"><a href="#" class="normal-link">Events <i class="fa fa-caret-down" aria-hidden="true"></i></a></div>
+                                <div class="inline-list"><a href="#" class="normal-link">Business<i class="fa fa-caret-down" aria-hidden="true"></i></a></div>
+                                <div class="inline-list"><a href="#" class="normal-link">AMAs</a></div>
+                                <div class="inline-list"><a href="#" class="normal-link">Spaces</a></div>
+                                <div class="inline-list"><a href="#" class="normal-link">Terms &amp; Conditions</a></div>
+                            </div>
+                            <div id="top-nav-links-right">
+                                <div class="inline-list">About</div>
+                                <div class="inline-list">Advertise</div>
+                                <div class="inline-list">Jobs</div>
+                                <div class="inline-list">Contact</div>
+                            </div>
+                        </div>
+                        <div id="bot-nav-links">
+                            <div class="bot-nav-text-links">
+                                <div class="bot-nav-link">Latest</div>
+                                <div class="bot-nav-link">Hard fork</div>
+                                <div class="bot-nav-link">Plugged</div>
+                                <div class="bot-nav-link">Readme</div>
+                                <div class="bot-nav-link">Growth quarters</div>
+                            </div>
+                            <div id="bot-nav-icons">
+                                <div class="header-icon" id="search-icon"><i class="fa fa-search" aria-hidden="true"></i></div>
+                                <div class="header-icon" id="menu-icon"><i class="fa fa-bars" aria-hidden="true"></i></div>
```

Comment:
- [ ] **These are also enclosed with an `<a>` tag**

Reply:
Duly Noted!
Thank you for your amazing feedback!

> # CHANGES REQUIRED 👀🤔
> ## Hey @notlfish @DanSam5K!!
> ### You've done a great job so far 🙌, but there are many things that we need to change and improve before this project can be approved. 😌
> ### First let's talk about your README.
> * **Your README is OK, but we need it to look more professional.**
>   
>   * [x]  Change the title for the current project.
>   * [x]  The README is not as descriptive as it can be. Add more to the description instead of just "Practice project for HTML.....". There are many words you can use to describe the type of work you are doing.
>   * [x]  Change live version link for the current project
>   * [x]  Add screenshots of your project
>   * **You can use the [template](https://github.com/microverseinc/readme-template) offered before to make your README more incredible.**
> 
> ### About your project
> * **Your project looks good, but there are things missing in your project. When you are cloning a website, try to make it look as similar as possible to the original. You can use the offered link to guide yourself.**
> 
> Original	Clone
> <img alt="image" width="320" src="https://user-images.githubusercontent.com/64392568/113079263-557a4680-919a-11eb-8ef5-f562502a1e24.png">	<img alt="image" width="320" src="https://user-images.githubusercontent.com/64392568/113079298-675be980-919a-11eb-8133-7198087d352e.png">
> * [x]  The logo is not the same
> * [x]  Text in the yellow container is bigger and have less padding in the clone
> * [x]  There is space between the yellow container and the articles compared to the original
> * [x]  There is less padding in the articles compared to the original
> 
> Original	Clone
> <img alt="image" width="493" src="https://user-images.githubusercontent.com/64392568/113079850-72fbe000-919b-11eb-855c-be40b9cab683.png">	<img alt="image" width="595" src="https://user-images.githubusercontent.com/64392568/113079937-97f05300-919b-11eb-9ea2-5199ec30ff23.png">
> * [x]  Remove logo in desktop resolution
> * [x]  Align the text in yellow container correctly
> 
> Original	Clone
> <img alt="image" width="1440" src="https://user-images.githubusercontent.com/64392568/113080080-e4d42980-919b-11eb-81a5-8d20eb9de4fd.png">	<img alt="image" width="1440" src="https://user-images.githubusercontent.com/64392568/113080109-f5849f80-919b-11eb-8d5c-d20c243c6fbe.png">
> * [x]  The first article is not aligned correctly
> * [x]  There is more whitespace between the two right articles compared to the original
> 
> Original	Clone
> <img alt="image" width="1440" src="https://user-images.githubusercontent.com/64392568/113080288-472d2a00-919c-11eb-97a3-81f5bd876a0b.png">	<img alt="image" width="1440" src="https://user-images.githubusercontent.com/64392568/113080342-63c96200-919c-11eb-90d9-d68a6708a35d.png">
> * [ ]  The Latest funding section is not aligned correctly
> 
> Original	Clone
> <img alt="image" width="1440" src="https://user-images.githubusercontent.com/64392568/113080485-9d9a6880-919c-11eb-842c-a325a161e28d.png">	<img alt="image" width="1440" src="https://user-images.githubusercontent.com/64392568/113080519-abe88480-919c-11eb-8996-520756a475cb.png">
> * [ ]  Items in the footer are not aligned correctly and some items are bigger than its supposed to be
> 
> ### Please try to finish the whole milestone before asking for a review, remember there is a limited number of reviews.
> ### SEE BELOW FOR MORE CHANGES 👇
> ### Keep the awesome work! 💪 Happy Coding for y'all!! 💻
> **If you have any questions you can ask me on my Slack.**
> 
> **Best,**
> 
> **[💁‍♂️ Steven Jack Chung Yau](https://stevenchung.me)**
> 
> [LinkedIn](https://linkedin.com/in/stevenjchung) | [Twitter](https://twitter.com/_yiak) | [GitHub](https://github.com/jcy2704)

duly Noted thank you for your amazing feedback!🙌